lldb/source/Symbol/Variable.cpp

```diff
@@ -438,6 +438,19 @@ Status Variable::GetValuesForVariableExpressionPath(
   return error;
 }
 
+bool Variable::IsThreadLocal() const {
+  ModuleSP module_sp(m_owner_scope->CalculateSymbolContextModule());
+  // Give the symbol vendor a chance to add to the unified section list.
+  module_sp->GetSymbolFile();
+  std::vector<uint32_t> symbol_indexes;
+  module_sp->GetSymtab()->FindAllSymbolsWithNameAndType(
```

Comment:
If you are going to do this code elsewhere, like in an OS ABI plug-in, please use `FindFirstSymbolWithNameAndType` if you only use the first one. Probably specify `lldb::SymbolType::eSymbolTypeData` as well